packages/pwa/app/partials/product-view/index.jsx

```diff
@@ -188,6 +192,12 @@ const ProductView = ({
         }
     }, [location.pathname])
 
+    useEffect(() => {
+        if (canOrder) {
+            toggleOptionsMessage(false)
+        }
+    })
```

Comment:
This effect runs for every component update, maybe we can optimize it a bit to only run when necessary. It seems to me that the only time the message should get updated is when the variant changes.

```suggestion
    useEffect(() => {
        if (canOrder) {
            toggleOptionsMessage(false)
        }
    }, [variant?.productId])
```